site2/docs/administration-pulsar-manager.md

```diff
@@ -0,0 +1,128 @@
+---
+id: administration-pulsar-manager
+title: The Pulsar Manager
```

Comment:
Do you need to delete the word `The`? http://pulsar.apache.org/docs/en/administration-dashboard/

Reply:
Yes. Do not add an article before a proper noun.

For example, we say "In 2013 Oracle introduced Oracle Database 12c—the world's first database designed for the cloud. " rather than "In 2013 Oracle introduced **the** Oracle Database 12c—the world...".